topdown/http_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2018 The OPA Authors.  All rights reserved.
```

Comment:
There's a bit of duplication in these tests. I think the setup and server could be factored out and then each test case would be quite concise. This is lower priority and not necessarily required before merging.